R-packages/evalcast/tests/testthat/test-validate_geo_type.R

```diff
@@ -1,23 +1,30 @@
 test_that("we can validate geo types", {
-  pct <- tibble(type_len = 2L, has_chars = TRUE)
-  expect_error(validate_geo_type("xyz", pct))
-  expect_equal(validate_geo_type("state", pct), "state")
-  expect_equal(validate_geo_type("county", pct), "state")
-  expect_error(validate_geo_type("hrr", pct))
-  pct <- tibble(type_len = 1:2, has_chars = FALSE)
-  expect_equal(validate_geo_type("hrr",pct), "hrr")
-  expect_equal(validate_geo_type("msa",pct), "msa")
-  expect_equal(validate_geo_type("dma",pct), "dma")
-  expect_error(validate_geo_type("state",pct))
-  expect_error(validate_geo_type("county",pct))
-  pct <- tibble(type_len = 5L, has_chars = FALSE)
-  expect_equal(validate_geo_type("county", pct), "county")
-  expect_equal(validate_geo_type("state", pct), "county")
-  expect_equal(validate_geo_type("hrr", pct), "hrr")
-  pct <- tibble(type_len = 1L, has_chars = TRUE)
-  expect_error(validate_geo_type("state",pct))
-  expect_error(validate_geo_type("county",pct))
-  pct <- tibble(type_len = 1L, has_chars = FALSE)
-  expect_error(validate_geo_type("state",pct))
-  expect_error(validate_geo_type("county",pct))
+  state_geos <- c("ak", "mn")
+  expect_error(validate_geo_type("xyz", state_geos))
```

Comment:
`expect_error` can take another argument to match the error message.  Would be stronger here to include the error message since there are multiple errors that could be triggered

Reply:
Tests removed since we are no longer doing the validation in evalcast.